CMake/resolve_dependency_modules/folly.cmake

```diff
@@ -44,10 +44,19 @@ if(COMPILER_HAS_W_UNDEF_PREFIX)
 endif()
 set(FOLLY_CXX_FLAGS -Wno-unused -Wno-unused-parameter -Wno-overloaded-virtual
                     ${EXTRA_CXX_FLAGS})
+
+if(gflags_SOURCE STREQUAL "BUNDLED")
+  set(glog_patch && git apply
+                 ${CMAKE_CURRENT_LIST_DIR}/folly/folly-gflags-glog.patch)
+endif()
+
 FetchContent_Declare(
   folly
   URL ${FOLLY_SOURCE_URL}
-  URL_HASH SHA256=${VELOX_FOLLY_BUILD_SHA256_CHECKSUM})
+  URL_HASH SHA256=${VELOX_FOLLY_BUILD_SHA256_CHECKSUM}
```

Comment:
I am curious, wont we have to update these patches when we change folly versions ?

Reply:
If there are significant changes to the cmake files yes but we are not changing anything intricate so it should be easy to update them in a folly clone and `git diff`